.chloggen/service-extension.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: collector
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: support for creating a service for extensions when ports are specified.
+
+# One or more tracking issues related to the change
+issues: []
```

Comment:
Missing issue id

Reply:
Created an issue and attached it in the chloggen.